integration-cli/docker_cli_by_digest_test.go

```diff
@@ -301,15 +233,10 @@ func (s *DockerRegistrySuite) TestListImagesWithDigests(c *check.C) {
 
 	// make sure repo shown, tag=<none>, digest = $digest1
 	re1 := regexp.MustCompile(`\s*` + repoName + `\s*<none>\s*` + digest1.String() + `\s`)
-	if !re1.MatchString(out) {
-		c.Fatalf("expected %q: %s", re1.String(), out)
-	}
-
+	c.Assert(re1.MatchString(out), checker.True, check.Commentf("expected %q: %s", re1.String(), out))
```

Comment:
checker.Matches doesn't work in this scenario (I tried messing around with it to make it work). Checkout how they are doing it here
- https://github.com/docker/docker/blob/2dd822f3226393b5611364785793d38a44debd08/integration-cli/docker_cli_import_test.go
- https://github.com/docker/docker/blob/eebf88ffd336819ecfdd7bc1206ee7bc8fbfe7ed/integration-cli/docker_cli_pull_test.go
- https://github.com/docker/docker/blob/c3f42b29a805e41a1e518653a1855c4e2a7f8d43/integration-cli/docker_cli_port_test.go